build.gradle

```diff
@@ -92,7 +102,7 @@ dependencies {
     // Dependencies that library users should include in their job
     // shadow jar
     // --------------------------------------------------------------
-    implementation "org.alvearie.hri:hri-api-batch-notification:develop-SNAPSHOT"
+    implementation "org.alvearie.hri:hri-api-batch-notification:WHFHRI-693-SNAPSHOT"
```

Comment:
I think you do NOT want to change this dependency here. you want it to pull in `develop-SNAPSHOT`.

I think using `develop-SNAPSHOT` should work the same as using `WHFHRI-693-SNAPSHOT`.

Reply:
I changed it again while troubleshooting again, it is now reverted ✅